### ISSUE TITLE
QTooltips briefly show before disappearing on Windows Phone

### ISSUE THREAD

**Issue by Sumurai8** [quasarframework/quasar#1099]
### Software version

Quasar: 0.14.7
OS: Windows Phone
Browsers: Native

### What did you get as the error?
When a lot of QTooltips in the application, they are briefly shown before disappearing from view. This looks... weird.

I presume this is because of this code, which will run only after all the QTooltips are created, combined with the absolutely aweful cpu of a windows phone.
```
    this.$nextTick(() => {
      /*
        The following is intentional.
        Fixes a bug in Chrome regarding offsetHeight by requiring browser
        to calculate this before removing from DOM and using it for first time.
      */
      this.$el.offsetHeight // eslint-disable-line
      this.anchorEl = this.$el.parentNode
      this.anchorEl.removeChild(this.$el)
```

### What were you expecting?
When navigating to a view with lots of QTooltips we don't want to see any of the until the user taps an anchor element with a tooltip.

### What steps did you take, to get the error?
Have a QDataTable with about 20-30 rows and tooltips in most of them. Navigate to that view and see how the QToolTips show up, then vanish.

**Comment by rstoenescu** [comment]
Will investigate. If anyone care to have a shot at it, pls do through a PR. Thanks for reporting!

**Comment by Sumurai8** [comment]
An easy workaround is to put a class on your component and put the following css in your component styling:

```
.component-container {
  .q-toolbar {
    display: none;
  }
}
```

Do you know which bug in Chrome that fix was for?

**Comment by rstoenescu** [comment]
This is fixed in 0.15.2.
